README.md

```diff
@@ -121,6 +121,14 @@ If you wish for new features or encounter any bug, please feel free to open [iss
 
 If you are reporting an issue, please generously turn on debug log with `export TAPE_LOGLEVEL=debug` and paste log in the issue
 
+## Contributor
```

Comment:
are we listing maintainer or contributor? (we have more than 4 contributors actually)